package io.urdego.group_service.common.exception.group;

public class GroupException {
}
